File: 6_Kyu_Kata/Multiples_of_3_or_5/Swift/MultiplesOf3Or5.swift
// 6 kyu - Multiples of 3 or 5  [ ID: 514b92a657cdc65150000006  (multiples-of-3-or-5) ]
// URL: https://www.codewars.com/kata/514b92a657cdc65150000006
// Category: undefined  |  Tags: MATHEMATICS | ALGORITHMS
// *****************************************************************************
func solution(_ num: Int) -> Int {
    var sum: Int = 0
    if num > 2 {
      for (i) in Array(0...(num-1)) {
        if (i % 3 == 0 || i % 5 == 0) {
          sum += i
        }
      }
    }
    return sum
  }

File: kata-6-kyu/multiples-of-3-or-5/python/multiples_of_3_or_5.py
# + =====================================================================================================================
# +
# + 6 kyu - Multiples of 3 or 5  [ ID: 514b92a657cdc65150000006 ] (multiples-of-3-or-5)
# + URL: https://www.codewars.com/kata/514b92a657cdc65150000006
# + Category: ALGORITHMS  |  Tags: MATHEMATICS | ALGORITHMS
# +
# + =====================================================================================================================


def solution(n):
    return sum(i for i in range(1, n) if i % 3 == 0 or i % 5 == 0)
